### PR TITLE
Worldcereal crop extent resampled

### PR DESCRIPTION
**Pull request by VincentVerelst**
Updated the worldcereal_crop_extent UDP to include a `projection` parameter, which allows the user to set the projection of the final results.

### CODE REVIEW

**Comment by VincentVerelst** [comment]
@jdries , the four failing unit tests are due to 4 application packages in the algorithm_catalog which are out of scope of this PR

File: openeo_udp/worldcereal_crop_extent.json [hunk 1]
```diff
@@ -1291,27 +1320,13 @@
       }
     },
     {
-      "name": "orbit_state",
-      "description": "The orbit state of the Sentinel-1 data",
+      "name": "projection",
+      "description": "The EPSG code of the output projection",
```
Comment (jdries) [inline openeo_udp/worldcereal_crop_extent.json:1324]: best to specify that UTM is the default, users should know that they do not need to set this value.

File: openeo_udp/worldcereal_crop_extent.json [hunk 2]
```diff
@@ -1291,27 +1320,13 @@
...
-          "DESCENDING"
-        ]
+        "type": "string"
       },
-      "default": "DESCENDING",
+      "default": null,
       "optional": true
     }
-  ],
-  "minimal_job_options": {
```
Comment (jdries) [inline openeo_udp/worldcereal_crop_extent.json:1307]: does it work without job options?

File: benchmark_scenarios/worldcereal_crop_extent.json [hunk 3]
```diff
@@ -36,12 +36,12 @@
...
-      "cropland-raw_2018-05-01_2019-04-30.tif": "https://s3.waw3-1.cloudferro.com/apex-benchmarks/worldcereal_crop_extent/cropland-raw_2018-05-01_2019-04-30.tif",
-      "job-results.json": "https://s3.waw3-1.cloudferro.com/apex-benchmarks/worldcereal_crop_extent/job-results.json",
-      "openEO.tif": "https://s3.waw3-1.cloudferro.com/apex-benchmarks/worldcereal_crop_extent/openEO.tif"
+      "worldcereal-cropland-extent.tif": "https://s3.waw3-1.cloudferro.com/swift/v1/APEx-benchmarks/worldcereal_crop_extent/worldcereal-cropland-extent.tif",
+      "job-results.json": "https://s3.waw3-1.cloudferro.com/swift/v1/APEx-benchmarks/worldcereal_crop_extent/job-results.json",
+      "worldcereal-cropland-extent-postprocessed.tif": "https://s3.waw3-1.cloudferro.com/swift/v1/APEx-benchmarks/worldcereal_crop_extent/worldcereal-cropland-extent-postprocessed.tif"
     },
     "reference_options": {
-      "atol": 1
+      "atol": 10
```
Comment (jdries) [inline benchmark_scenarios/worldcereal_crop_extent.json:44]: isn't this a rather high tolerance for a discrete classification?

**Review by jdries — CHANGES_REQUESTED** [review]

File: openeo_udp/worldcereal_crop_extent.json (hunk 2 above)
Comment (VincentVerelst) [reply openeo_udp/worldcereal_crop_extent.json:1307]: Currently, the only way it works, is to specify job options after calling the UDP

File: benchmark_scenarios/worldcereal_crop_extent.json (hunk 3 above)
Comment (VincentVerelst) [reply benchmark_scenarios/worldcereal_crop_extent.json:44]: Yes, so here we have a bit of a problem:
- The benchmarks compare all bands of the geotiff at once with one single tolerance level
- For the actual classification (0 or 1) this tolerance indeed doesn't make much sense, but also the classification probabilities are included in the geotiffs. These seemed to vary quite a lot, causing the benchmark to always fail with an absolute tolerance of 1

**Pull request merged**